src/playground-ide.ts

```diff
@@ -153,9 +153,60 @@ export class PlaygroundIde extends LitElement {
 
   /**
    * A document-relative path to a project configuration file.
+   *
+   * When both `projectSrc` and `files` are set, the one set most recently wins.
+   * Slotted children win only if both `projectSrc` and `files` are undefined.
    */
-  @property({attribute: 'project-src'})
-  projectSrc?: string;
+  @property({attribute: 'project-src', hasChanged: () => false})
```

Comment:
can you add a comment on the `hasChanged: () => false` option?

Reply:
Done

```
// Note we set `hasChanged: () => false` because we don't need to trigger
// `update` when this property changes. (Why be a lit property at all?
// Because we want [1] to respond to atribute changes, and [2] to inherit
// property values set before upgrade)
```